### ISSUE TITLE
model from path 16-bits training:True but float16 false

### ISSUE THREAD

**Issue by mahdirezaey** [huggingface/transformers#4105]
# ❓ Questions & Help

## Details

**A link to original question on Stack Overflow**:

**Comment by mahdirezaey** [comment]
Hi all

I am gonna run "run_language_modeling.py" on 1 GPU 1080 Ti  
and loading distill bert from directory 

and using fp16 , apex

it is run OK 
and i am getting trace back of 


05/01/2020 22:24:38 - WARNING - __main__ -   Process rank: -1, device: cuda, n_gpu: 1, distributed training: False, 16-bits training: store_true
05/01/2020 22:24:38 - INFO - transformers.configuration_utils -   loading configuration file ./save_pre_trn_model/config.json
05/01/2020 22:24:38 - INFO - transformers.configuration_utils -   Model config DistilBertConfig {
  "_num_labels": 2,
  "activation": "gelu",
  "architectures": [
    "DistilBertModel"
  ],
  "attention_dropout": 0.1,
  "bad_words_ids": null,
  "bos_token_id": null,
  "decoder_start_token_id": null,
  "dim": 768,
  "do_sample": false,
  "dropout": 0.1,
  "early_stopping": false,
  "eos_token_id": null,
  "finetuning_task": null,
  "hidden_dim": 3072,
  "id2label": {
    "0": "LABEL_0",
    "1": "LABEL_1"
  },
  "initializer_range": 0.02,
  "is_decoder": false,
  "is_encoder_decoder": false,
  "label2id": {
    "LABEL_0": 0,
    "LABEL_1": 1
  },
  "length_penalty": 1.0,
  "max_length": 20,
  "max_position_embeddings": 512,
  "min_length": 0,
  "model_type": "distilbert",
  "n_heads": 12,
  "n_layers": 6,
  "no_repeat_ngram_size": 0,
  "num_beams": 1,
  "num_return_sequences": 1,
  "output_attentions": false,
  "output_hidden_states": false,
  "output_past": true,
  "pad_token_id": 0,
  "prefix": null,
  "pruned_heads": {},
  "qa_dropout": 0.1,
  "repetition_penalty": 1.0,
  "seq_classif_dropout": 0.2,
  "sinusoidal_pos_embds": false,
  "task_specific_params": null,
  "temperature": 1.0,
  "tie_weights_": true,
  "top_k": 50,
  "top_p": 1.0,
  "torchscript": false,
  "use_bfloat16": false,
  "vocab_size": 30000
}

**Comment by mahdirezaey** [comment]
Is it OK to see 
16-bits training:True   (i have written "store_true" in the code of it instead of True but those are doing the same thing)
and
"use_bfloat16": false

with each other ?

what does "use_bfloat16": false means ?

**Comment by BramVanroy** [comment]
[As far as I can see](https://github.com/huggingface/transformers/search?q=use_bfloat16&type=Code) `bfloat16` is only relevant for Tensorflow XLNet, so no need to worry.
